src/Components/Carousel/Carousel.js

```diff
@@ -35,22 +60,30 @@ const Carousel = () => {
      }
 
     const goBack = () => {
+        if ( offset > 0) {
         setOffset(offset - count)
+        } else {
+        setOffset(offset)
+        }
         }
     
-
     const goForward = () => {
+        if ( offset < numberReq - count) {
         setOffset(offset + count)
-    }
+        }
```

Comment:
Code formatting should be improved.
Check out this article https://medium.com/@pgivens/write-cleaner-code-using-prettier-and-eslint-in-vscode-d04f63805dcd